css/styles.css

```diff
@@ -81,6 +93,19 @@ div.recipe-card {
   cursor: pointer;
 }
 
+section.result-container {
+  width: 100%;
+  height: 100%;
+  overflow-y: scroll;
```

Comment:
Most of the time, `auto` is more useful than `scroll`, since it only shows a scrollbar if there is, in fact, scrolling.